.github/workflows/build.yml

```diff
@@ -3,7 +3,14 @@ on: push
 
 jobs:
     build:
-        runs-on: ubuntu-latest
+        runs-on: ${{ matrix.os }}
+        strategy:
+            matrix:
+                os: [ubuntu-latest, windows-latest]
+        defaults:
+            run:
+                shell: ${{ matrix.os == 'windows-latest' && 'cmd' || 'bash' }}
```

Comment:
On windows the default shell is apparently powershell and it does not make the build fail on non zero exit codes...